native/cocos/audio/android/AudioEngine-inl.cpp

```diff
@@ -405,6 +405,11 @@ PCMHeader AudioEngineImpl::getPCMHeader(const char *url) {
         CC_LOG_DEBUG("file %s does not exist or failed to load", url);
         return header;
     }
+    if (_audioPlayerProvider->getPcmHeader(url, header)) {
+        CC_LOG_DEBUG("file %s pcm data already cached", url);
+        return header;
```

Comment:
When an error occurs a PCMHeader will still be passed to the caller, I suppose a success code should return here? Otherwise, the caller can not tell if the result is correct.

Reply:
Make sense, yet it's not the key to solving this issue, I will make it later.